WordPress/src/main/java/org/wordpress/android/ui/jetpackoverlay/JetpackFeatureRemovalPhaseHelper.kt

```diff
@@ -126,6 +131,26 @@ class JetpackFeatureRemovalPhaseHelper @Inject constructor(
         }
     }
 
+    @JvmOverloads
+    fun trackPageAccessedEventIfNeeded(pageType: PageType, site: SiteModel? = null) {
+        when (pageType) {
+            PageType.MY_SITE -> analyticsTrackerWrapper.track(AnalyticsTracker.Stat.MY_SITE_ACCESSED, site)
+            PageType.READER -> {
+                if (!shouldShowStaticPage()) {
+                    analyticsTrackerWrapper.track(AnalyticsTracker.Stat.READER_ACCESSED)
+                }
+            }
+
+            PageType.NOTIFS -> {
+                if (!shouldShowStaticPage()) {
```

Comment:
Tested and works as expected. 

Nitpick: Maybe would be nice to include a function  that inversely wraps the `!shouldShowStaticPage()` call. Something like `private fun shouldTrackDynamicPage() = !shouldShowStaticPage()`.

Anyway, feel free to merge this!

Reply:
I was initially doing something like `false == shouldShowStaticPage()`, but AS was raising warning to simplify with the `!` and went with that 😄 ... but makes sense 👍 , changed here 36b49c2c7fd6b6e854890cd5effd74a780bbdbc9